src/provider/view/containers/NotificationCenterApp/NotificationsFeedsPanel/FeedsPanel.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import {connect} from 'react-redux';
+import {RouteComponentProps} from 'react-router';
+import {MemoryHistory} from 'history';
+
+import {RootState} from '../../../../store/State';
+import {Header} from '../../../components/Header/Header';
+import {ClassNameBuilder} from '../../../utils/ClassNameBuilder';
+import {FeedsView} from '../../../components/FeedsView/FeedsView';
+
+import * as Styles from './FeedsPanel.module.scss';
+
+interface Props extends RouteComponentProps {
+
+}
+
+const FeedsPanelComponent: React.FC<Props> = (props) => {
```

Comment:
Props should define `history: MemoryHistory`; Then you don't need to cast or use RouteComoponentProps as that is the wrong try of history.

Reply:
Though i had changed all of them, but must have missed one. Will fix now.